website/docs/docs/integrate/avail-sl-integrations.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Available integrations"
+id: avail-sl-integrations
+description: "Review a wide range of partners you can integrate and query with the dbt Semantic Layer."
+sidebar_label: "Available integrations"
+---
+
+# Available integrations
+
+The dbt Semantic Layer brings a wide range of partners you can integrate and query with, including BI tools, Notebooks, ML models, and Data apps. Get ready to extend the analytics engineering workflow beyond dbt Cloud, with seamless integrations covering a range of use cases across the Modern Data Stack. 
+
+There are two categories of partner integrations: 
+
+-  **dbt Semantic Layer partners** - This refers to a group of dbt Labs partners who have full end-to-end integrations with the dbt Semantic Layer and leverage the dbt Server. 
+
+-  **dbt metrics ready partners** - This refers to all partners integrating with dbt metrics functionality but do not leverage the dbt Server.
```

Comment:
Should be "Metrics Ready partners"

Reply:
Recommend: "Metrics Ready integrations facilitate building, discovery, and collaboration on dbt metric definitions."

That matches the copy we're using on this landing page, and gives it a slightly more positive framing:
https://www.getdbt.com/product/semantic-layer-integrations